hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/AbstractMessageTask.java

```diff
@@ -90,17 +91,17 @@ public void run() {
                 handleMissingEndpoint();
                 return;
             }
-            //process message
+            // process message
             if (!node.joined()) {
                 throw new HazelcastInstanceNotActiveException("Hazelcast instance is not ready yet!");
             }
-            final Credentials credentials = endpoint.getCredentials();
+            Credentials credentials = endpoint.getCredentials();
             interceptBefore(credentials);
             checkPermissions(endpoint);
             processMessage();
             interceptAfter(credentials);
-
         } catch (Throwable e) {
+            logProcessingFailure(e);
```

Comment:
We may catch the `QueryResultSizeExceededException` here and suppress the logging of the stack trace.